src/main/java/com/box/sdk/BoxTransactionalAPIConnection.java

```diff
@@ -0,0 +1,166 @@
+    }
+
+    /**
+     * Disabling an invalid constructor for Box Developer Edition.
+     * @param  clientID     the client ID to use when requesting an access token.
+     * @param  clientSecret the client secret to use when requesting an access token.
+     * @throws UnsupportedOperationException Box Transactional API does not support creating a
+     *         connection with these params
+     */
+    public BoxTransactionalAPIConnection(String clientID, String clientSecret) {
+        super(null);
+        throw new UnsupportedOperationException("This constructor is not available for BoxTransactionalAPIConnection.");
+    }
+
+    /**
+     * Request a scoped transactional token.
+     * @param accessToken application access token.
+     * @param scope scope of transactional token.
+     * @return a BoxAPIConnection which can be used to perform transactional requests.
+     */
+    public static BoxAPIConnection getTransactionConnection(String accessToken, String scope) {
+        return BoxTransactionalAPIConnection.getTransactionConnection(accessToken, scope, null);
+    }
+
+    /**
+     * Request a scoped transactional token for a particular resource.
+     * @param accessToken application access token.
+     * @param scope scope of transactional token.
+     * @param resource resource transactional token has access to.
+     * @return a BoxAPIConnection which can be used to perform transactional requests.
+     */
+    public static BoxAPIConnection getTransactionConnection(String accessToken, String scope, String resource) {
+        BoxAPIConnection apiConnection = new BoxAPIConnection(accessToken);
+
+        URL url;
+        try {
+            url = new URL(apiConnection.getTokenURL());
+        } catch (MalformedURLException e) {
+            assert false : "An invalid token URL indicates a bug in the SDK.";
+            throw new RuntimeException("An invalid token URL indicates a bug in the SDK.", e);
+        }
+
+        String urlParameters;
+        try {
+            urlParameters = String.format("grant_type=%s&subject_token=%s&subject_token_type=%s&scope=%s", GRANT_TYPE,
+                    accessToken, SUBJECT_TOKEN_TYPE, URLEncoder.encode(scope, "UTF-8"));
+
+            if (resource != null) {
+                urlParameters += "&resource=" + URLEncoder.encode(resource, "UTF-8");
+            }
+        } catch (UnsupportedEncodingException e) {
+            throw new BoxAPIException(
+                    "An error occurred while attempting to encode url parameters for a transactional token request"
+            );
+        }
+
+        BoxAPIRequest request = new BoxAPIRequest(apiConnection, url, "POST");
+        request.shouldAuthenticate(false);
+        request.setBody(urlParameters);
+
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        JsonObject responseJSON = JsonObject.readFrom(response.getJSON());
```

Comment:
ParseException - if the input is not valid JSON
java.lang.UnsupportedOperationException - if the input does not contain a JSON object

Need to catch